ios/swift/AmpliSwiftSampleApp/Shared/ContentView.swift

```diff
@@ -0,0 +1,124 @@
+        NavigationView {
+            List {
+                ForEach(items) { item in
+                    NavigationLink {
+                        Text("Item at \(item.timestamp!, formatter: itemFormatter)")
+                    } label: {
+                        Text(item.timestamp!, formatter: itemFormatter)
+                    }
+                }
+                .onDelete(perform: deleteItems)
+            }
+            .toolbar {
+#if os(iOS)
+                ToolbarItem(placement: .navigationBarTrailing) {
+                    EditButton()
+                }
+#endif
+                ToolbarItem {
+                    Button(action: addItem) {
+                        Label("Add Item With Event Sent", systemImage: "plus")
+                    }
+                }
+            }
+            Text("Select an item")
+        }
+    }
+
+    private func addItem() {
+        //    Start by calling ampli.load()
+        //
+        //    'ampli' is the default instance of Ampli()
+        //
+        //    When you pull your tracking plan you can use the defaults and call load() without arguments
+        //    This requires connecting your account via `ampli pull` which will set you API key in the generated Ampli SDK
+        //
+        //    OR Specify a AmpliEnvironment
+        //    ampli.load(options: LoadOptions(environment: AmpliEnvironment.development)
+        //
+        //    OR Provide a specific Amplitude API key
+        //    ampli.load(options: LoadOptions(client: LoadClientOptions(apiKey: "Custom api key"))
+        //
+        //    OR Use an existing Amplitude instance
+        //    requires "import Amplitude"
+        //    let instance = Amplitude.instance("instanceName");
+        //    instance.initializeApiKey("Custom api key");
+        //    ampli.load(options: LoadOptions(client: LoadClientOptions(instance: instance)))
+        //
+        //    For testing you can disable ampli
+        //    ampli.load(options: LoadOptions(disabled: ENV.IS_TESTING ? true: false))
+        //
+        //    Make as many Ampli instances as you want
+        //    let ampli2 = new Ampli();
+        //    ampli2.load(options: LoadOptions(client: LoadClientOptions(apiKey: "api-key-2")))
+        let apiKey = ProcessInfo.processInfo.environment["AMPLITUDE_API_KEY"];
+        ampli.load(options: LoadOptions(client: LoadClientOptions(apiKey: apiKey)));
+        ampli.eventNoProperties(extra: nil)
+        let extraDict = ["test" : "extra test"];
+        ampli.eventMaxIntForTest(properties: EventMaxIntForTestProperties(intMax10: 20), extra: extraDict);
+        ampli.eventWithConstTypes(extra: extraDict)
+        ampli.track(event: EventWithAllProperties(eventProperties: EventWithAllPropertiesProperties(optionalString: nil, requiredArray: ["array element 1", "array element 2"], requiredBoolean: true, requiredEnum: RequiredEnum.enum1, requiredInteger: 10, requiredNumber: 2.0, requiredString: "required string")), extra: nil)
```

Comment:
Is extra optional here? If not can you please make it optional.

For this if you are trying to demonstrate the usage with extra then maybe set `extra: extraDict`